src/hardware/gus.cpp

```diff
@@ -1433,11 +1437,26 @@ void Gus::WriteToRegister()
 	return;
 }
 
+Gus::~Gus()
+{
+	DEBUG_LOG_MSG("GUS: Shutting down");
+	StopPlayback();
+
+	// remove the mixer channel
+	audio_channel.reset();
+
+	// remove the IO handlers
+	for (auto & rh : read_handlers)
+		rh.Uninstall();
+	for (auto & wh : write_handlers)
+		wh.Uninstall();
```

Comment:
I believe clang-formatter will correct these two to `auto &rh`?

Reply:
Yup, got'em. 
thanks.